toolchain/crosstool-ng/crosstool.config

```diff
@@ -515,55 +542,55 @@ CT_GLIBC_DEP_KERNEL_HEADERS_VERSION=y
 CT_GLIBC_DEP_BINUTILS=y
 CT_GLIBC_DEP_GCC=y
 CT_GLIBC_DEP_PYTHON=y
+CT_THREADS="nptl"
 CT_GLIBC_BUILD_SSP=y
 CT_GLIBC_HAS_LIBIDN_ADDON=y
 CT_GLIBC_USE_LIBIDN_ADDON=y
 CT_GLIBC_NO_SPARC_V8=y
-CT_GLIBC_HAS_OBSOLETE_RPC=y
 CT_GLIBC_EXTRA_CONFIG_ARRAY=""
 CT_GLIBC_CONFIGPARMS=""
+CT_GLIBC_ENABLE_DEBUG=y
```

Comment:
I guess this is default, and can be disabled, if preferred.

Reply:
I believe it should be disabled as this is an embedded system.

Reply:
Fair enough, disabled